tox.ini

```diff
@@ -8,11 +8,15 @@ deps =
     django111: django-formtools>=2.1,<2.2
     django21: Django>=2.1,<2.2
     django22: Django>=2.2,<2.3
-    cms36: https://github.com/divio/django-cms/archive/release/3.6.x.zip
-    cms37: https://github.com/divio/django-cms/archive/release/3.7.x.zip
-    cmsdev: https://github.com/divio/django-cms/archive/develop.zip
+    cms40: https://github.com/divio/django-cms/archive/release/4.0.x.zip
     -r{toxinidir}/requirements-test.txt
 
+basepython =
+    py27: python2.7
+    py35: python3.5
+    py36: python3.6
+    py37: python3.7
+
 [testenv:isort]
 commands = isort -c -rc
 deps = isort
```

Comment:
For the test suites that are failing with interpreter not found. Ie `isort`, `pep8` and `docs`. you need to add `basepython = python3.5` in other words it would look like the following. 

```
[testenv:isort]
commands = isort -c -rc
deps = isort
skip_install = true
basepython = python3.5
```

Reply:
It might be that you need a different basepython other than 3.5, if it doesn't work with 3.5, try with 3.6.